src/ert/config/queue_config.py

```diff
@@ -95,11 +73,14 @@ def from_dict(cls, config_dict: ConfigDict) -> QueueConfig:
                     " usually provided by the site-configuration file, beware that"
                     " you are effectively replacing the default value provided."
                 )
-        if (
-            selected_queue_system == QueueSystem.TORQUE
-            and queue_options[QueueSystem.TORQUE]
-        ):
-            _validate_torque_options(queue_options[QueueSystem.TORQUE])
+
+        for queue_system_val in used_queue_systems:
```

Comment:
Can we instead of `used_queue_systems` do  something like `queue_options.keys()`?